src/GovUk.Education.ExploreEducationStatistics.Admin/Services/UserManagementService.cs

```diff
@@ -212,15 +223,35 @@ public async Task<List<UserViewModel>> ListPreReleaseUsersAsync()
                         return ValidationActionResult(UserDoesNotExist);
                     }
 
-                    var userReleaseRoles = await GetUserReleaseRoles(user.Id);
+                    var userReleaseRoles = _contentDbContext.UserReleaseRoles
+                        .Include(urr => urr.Release)
+                        .ThenInclude(r => r.Publication)
+                        .ToList()
+                        .Where(urr => urr.UserId == Guid.Parse(userId) &&
+                                      IsLatestVersionOfRelease(urr.Release.Publication.Releases, urr.Release.Id))
```

Comment:
Just wondering if it might make sense to make Publication#IsLatestVersionOfRelease public and use that, e.g. `urr.Release.Publication.IsLatestVersionOfRelease(urr.Release.Id)` or add this as a new method to `ReleaseExtensions` as we seem to be doing this in a lot of places`. That would remove the helper method at the bottom of this class.

Reply:
As discussed, I've made Publication.IsLatestVersionOfRelease public and used that so I can remove UserManagementService#IsLatestVersionOfRelease. But, as Ben notes, the same IsLatestVersionOfRelease is all over the codebase so I'll create a separate ticket to refactor this more widely: EES-1796.